dependencies/pom.xml

```diff
@@ -510,10 +510,25 @@
                 <artifactId>tyrus-client</artifactId>
                 <version>${version.lib.tyrus}</version>
             </dependency>
+            <dependency>
+                <groupId>org.glassfish.tyrus</groupId>
+                <artifactId>tyrus-container-grizzly-client</artifactId>
```

Comment:
Is this dependency meant to be used by end-users ? If not they can be moved to the top level pom.

Reply:
They would use it only for testing WS, just like we do now. WDYT?

Reply:
In that case, that is the right place for this.